### DISCUSSION TITLE
Rest API Data Provider Filtering not working

### DISCUSSION THREAD

**Discussion by Yacobolo** [Budibase/budibase#7725] [Help]
When i add a filter to the data provider it does not return any rows. When i change the data from query (Rest API) to table (BudiDB) it works. Am i doing anything wrong, or is it not possible to filter a query?

Note: The Dynamic filter and the tableblock search/filtering works

<img width="620" alt="image" src="https://user-images.githubusercontent.com/45033225/189543644-865c9b36-b921-4482-a82a-d2f39d3f5350.png">

<img width="646" alt="image" src="https://user-images.githubusercontent.com/45033225/189543679-fa9ba558-cf98-4243-ba7c-a4b18cc0a0e2.png">

<img width="411" alt="image" src="https://user-images.githubusercontent.com/45033225/189543699-3856b728-7a0b-40ca-a7ff-7122efdea3b8.png">

**Comment by melohagan** [comment]
Hi @Yacobolo 

I'm not seeing any issues. Could you share an example payload returned by your API please?
